.rubocop.yml

```diff
@@ -17,6 +20,9 @@ Layout/EndAlignment:
 Layout/HeredocIndentation:
   Enabled: false
 
+Layout/LineLength:
+  Max: 120
+
```

Comment:
Is this just a personal preference? Or is there an additional reason for making this change?

Reply:
From RuboCop v0.84 onward, the default max length is 120. Now that we're still using RuboCop v0.81 because of MRI 2.3 support, I added this because of consistency reasons 😄 
As soon as you decide to drop support for 2.3, this can be removed as well!

Reply:
I actually have 80 chars set as my [personal preference](https://gist.github.com/pat/ba3b8ffb1901bfe5439b460943b6b019/#file-rubocop-yml-L17) - so, would prefer to stick with that even if Rubocop's default has changed.

Reply:
Oh absolutely! Done in the latest push! 😄